nnstreamer/tensor_filter/meson.build

```diff
@@ -24,7 +24,7 @@ shared_library('nnstreamer_filter_nntrainer',
   dependencies: nnstreamer_filter_nntrainer_deps,
   include_directories: [nntrainer_inc, '.'], # '.' shouldn't be installed
   install: true,
-  install_dir: filter_subplugin_install_dir
+  install_dir: '/usr/lib/nnstreamer'
```

Comment:
Keep this configurable (w/ meson_options?).
If a user wants to support multilib, there could be multiple instances of libnnstreamer_filter_nntrainer.so in a single device.

Reply:
Okay updated :)